### DISCUSSION TITLE
car icon

### DISCUSSION THREAD

**Discussion by LuisIntralix** [tailwindlabs/heroicons#687] [Ideas]
Please make a car / truck icon. Thanks!

**Comment by stevecastaneda** [comment]
Truck icon is available, but +1 to a car!

<img width="149" alt="CleanShot 2023-01-03 at 16 15 51@2x" src="https://user-images.githubusercontent.com/171488/210450229-a04b44af-f1bc-4724-bac9-b2aa78eef1ce.png">
